### PR TITLE
add parallel-hashmap/1.31

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **parallel-hashmap/1.31**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/parallel-hashmap/all/conandata.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.31":
+    url: "https://github.com/greg7mdp/parallel-hashmap/archive/1.31.tar.gz"
+    sha256: "1826589041140837f91ab0e3421abbf7e9184454bb83403b5a0359a0bf87bd68"
   "1.30":
-    url: "https://github.com/greg7mdp/parallel-hashmap/archive/1.30.zip"
-    sha256: "388da5500842b42ea9f643220541aa13c6b1e0c810ecf603d65e174fe3eb1eee"
+    url: "https://github.com/greg7mdp/parallel-hashmap/archive/1.30.tar.gz"
+    sha256: "ae14fa41110b5fe211d3f7599318fb1a3e862353c7110cd8370cdc044a98d815"
```
Comment (SSE4) [inline recipes/parallel-hashmap/all/conandata.yml:7]: thanks for changing to tgz, it's always better than zip

**Comment by SpaceIm** [comment]
@jsgogo @danimtb 

`parallel-hashmap/1.31` can't be found either through `conan search` or in https://conan.io/center

**Comment by gocarlos** [comment]
@SpaceIm you mean @jgsogo

**Comment by SpaceIm** [comment]
Ooops yes :)

**Pull request merged**